Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
```

Comment:
### [sidekiq](https://sidekiq.org)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 6.4.1 → 6.5.6

[_[change-log](https://github.com/mperham/sidekiq/blob/main/Changes.md), [source-code](https://github.com/mperham/sidekiq), [gem-diff](https://my.diffend.io/gems/sidekiq/6.4.1/6.5.6)_]





<details>
<summary>Commits</summary>

A change of **164** commits. See the full changes on [the compare page](https://github.com/mperham/sidekiq/compare/v6.4.1...v6.5.6).

These are the first 10 commits:
- (e59f1e5) [Cleanups for sidekiqload (#5136)](https://github.com/mperham/sidekiq/commit/e59f1e5d2dbbdb8215a9eafb3465da060fe053ba)
- (a150c7a) [chhhanges](https://github.com/mperham/sidekiq/commit/a150c7a8bb930c6d952d78318b133a8b20d884b5)
- (dce4b74) [oops, wait_readable misuse, fixes #5190](https://github.com/mperham/sidekiq/commit/dce4b747f7d86e38d8be3879e359b376786c9804)
- (eab794f) [Do not fix the width of the navigation status bar (#5191)](https://github.com/mperham/sidekiq/commit/eab794f17c175642ecaa847fee8c4ffd22c81105)
- (5bfaeea) [Always use String keys with setter hash, fixes #5192](https://github.com/mperham/sidekiq/commit/5bfaeea73ad939f95dfa362c1f939e6168c590cb)
- (0020086) [Exit sidekiqmon with non-zero exit code if an error occurs (…](https://github.com/mperham/sidekiq/commit/00200869ee8953212ff926a8affcadd64ceb58d5)
- (7cd97b4) [Remove parallel code paths caused by "push" and "push_bulk" …](https://github.com/mperham/sidekiq/commit/7cd97b4253b3e7a4f928b0af8719c6acaecedb44)
- (6a6fda3) [Don’t document this](https://github.com/mperham/sidekiq/commit/6a6fda35fa9d07c4fe8ed3b05609a0eba6781101)
- (091a77f) [Add perform_sync alias, fixes #5208](https://github.com/mperham/sidekiq/commit/091a77f4b0136faf4e230620056a2b65c017029c)
- (3b9058c) [#5211](https://github.com/mperham/sidekiq/commit/3b9058cf6388c7857cd094fbb400cdd48850e74d)


</details>